frontend/src/modules/entitydetails/components/TermsPopup.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
```

Comment:
Why is this file not in `modules/terms`?

Reply:
I've put the files specific for the Terms panel into modules/terms (we use the same logic for the history, machinery, otherlocales and teamcomments panels). Since the popup is not part of the panel, I've put it in modules/entitydetails, where it belongs UI-wise.